js/components/organization/card-filter.vue

```diff
@@ -13,9 +13,9 @@
                 <input type="text" name="search" class="form-control"
                     :placeholder="_('Search')" v-model="search_query" />
                 <div class="input-group-btn">
-                    <button type="submit" name="submit" class="btn btn-warning btn-flat">
+                    <div class="btn btn-warning btn-flat">
```

Comment:
Why did you remove the `button`?

Reply:
The button is just reloading the page and you lose all your search criteria when you click on it, it does nothing as the search happens when you are typing so no need for a submit button. ( Users get confused when they click on it, the page reloads and the search gets lost.)